resources/js/features/game-list/components/DataTableToolbar/DataTableDesktopToolbar.tsx

```diff
@@ -38,32 +42,52 @@ export function DataTableDesktopToolbar<TData>({
 
   const { t } = useTranslation();
 
+  const [isCurrentlyPersistingView, setIsCurrentlyPersistingView] = useAtom(
+    isCurrentlyPersistingViewAtom,
+  );
+
   const allColumns = table.getAllColumns();
 
   const currentFilters = table.getState().columnFilters;
   const isFiltered = getAreNonDefaultFiltersSet(currentFilters, defaultColumnFilters);
 
   return (
     <div className="flex w-full flex-col justify-between gap-2">
-      <div className="flex flex-col gap-2 rounded bg-embed p-2 sm:flex-row sm:gap-2 md:gap-3">
-        {doesColumnExist(allColumns, 'system') && filterableSystemOptions?.length > 1 ? (
-          <DataTableSystemFilter table={table} filterableSystemOptions={filterableSystemOptions} />
-        ) : null}
-
-        {doesColumnExist(allColumns, 'achievementsPublished') ? (
-          <DataTableAchievementsPublishedFilter table={table} />
-        ) : null}
-
-        {doesColumnExist(allColumns, 'progress') ? <DataTableProgressFilter table={table} /> : null}
-
-        {isFiltered ? (
-          <DataTableResetFiltersButton
-            table={table}
-            defaultColumnFilters={defaultColumnFilters}
-            tableApiRouteName={tableApiRouteName}
-            tableApiRouteParams={tableApiRouteParams}
+      <div className="flex w-full flex-col items-center justify-between gap-3 rounded bg-embed py-2 pl-2 pr-3 sm:flex-row">
+        <div className="flex w-full flex-col gap-2 sm:w-auto sm:flex-row md:gap-3">
+          {doesColumnExist(allColumns, 'system') && filterableSystemOptions?.length > 1 ? (
+            <DataTableSystemFilter
+              table={table}
+              filterableSystemOptions={filterableSystemOptions}
+            />
+          ) : null}
+
+          {doesColumnExist(allColumns, 'achievementsPublished') ? (
+            <DataTableAchievementsPublishedFilter table={table} />
+          ) : null}
+
+          {doesColumnExist(allColumns, 'progress') ? (
+            <DataTableProgressFilter table={table} />
+          ) : null}
+
+          {isFiltered ? (
+            <DataTableResetFiltersButton
+              table={table}
+              defaultColumnFilters={defaultColumnFilters}
+              tableApiRouteName={tableApiRouteName}
+              tableApiRouteParams={tableApiRouteParams}
+            />
+          ) : null}
+        </div>
+
+        <BaseLabel className="flex items-center gap-2 text-menu-link">
+          <BaseCheckbox
+            checked={isCurrentlyPersistingView}
+            onCheckedChange={(checked: boolean) => setIsCurrentlyPersistingView(checked)}
           />
-        ) : null}
+
+          {t('Remember my view')}
```

Comment:
While this does remember the view, the filters don't reflect the remembered values when returning to the page.
![image](https://github.com/user-attachments/assets/4fc912a5-3763-49b3-8e0f-53504afce8e9)

![image](https://github.com/user-attachments/assets/3e69a535-5452-4654-8f99-a414fb72f558)

Reply:
Should be fixed in latest.